docs/getting-started/how-to-create-custom-highcharts-files.md

```diff
@@ -95,78 +42,47 @@ export default Highcharts;
 ```
 
 Modify the setup according to your needs, then proceed with the next step.
-Please note that the order of the imports has to match the dependencies of each
-file. Optional additions come therefor last.
+Please note that the order of the imported ES modules should match the
+dependencies of each file. Optional additions come therefor last.
 
+For similar examples take a look at the other master files.
 
 
-Configure and run the Highcharts assembler
-------------------------------------------
 
-Create a new file `./custom-builder.js`. Add the following configuration to the
-file:
-
-```js
-'use strict';
-/**
- * Assembler is installed as an NPM package.
- * See "Install the Highcharts assembler" for more information.
- */
-const build = require('highcharts-assembler');
-build({
-    base: './highcharts/js/masters/',
-    files: ['custom.src.js'],
-    output: './dist/'
-});
-```
+Create the custom package file
+------------------------------
 
-Open a CLI in your project folder and run `node custom-builder.js` to execute
-the build script. Once the script has completed you should find the resulting
-script `./dist/custom.src.js`.
+Run `npm test` to build all package files out of the master files and test
```

Comment:
Running `npm test` will also run the whole test suite, which is slow and boring if users only want to change some minor thing. I would rather say "Run `npm scripts` to build all package files out of the master files. For running the automated tests in addition, run `npm test`".